src/video/windows/SDL_windowswindow.c

```diff
@@ -236,14 +241,15 @@ static void WIN_AdjustWindowRect(SDL_Window *window, int *x, int *y, int *width,
     WIN_AdjustWindowRectWithStyle(window, style, menu, x, y, width, height, use_current);
 }
 
-void WIN_SetWindowPositionInternal(SDL_Window *window, UINT flags)
+int WIN_SetWindowPositionInternal(SDL_Window *window, UINT flags)
```

Comment:
I'm not happy with the changes in this function.
Please provide suggestions to improve the quality.